index.html

```diff
@@ -596,14 +547,95 @@ <h2>Internationalization</h2>
 <section>
   <h3>Appendix</h3>
   <section>
-      <h3>JSON Examples</h3>
+    <h2>Controllers</h2>
+        <pre class="example" title="A verifiable credential controller document">
```

Comment:
This example seems out of place. What's the intent of putting a DID document example in `vc-jwt`?

Reply:
Its necessary to explain controller documents, because they are related to `iss` and `kid`, similar to:

https://w3c.github.io/vc-data-integrity/#controller-documents

Reply:
Just read that section, and I have to say that it's not intuitive at all. I think it would be worth either: adding an example that refers to this controller document, pointing to the spec you mentioned, or explicitly stating the relationship between controller docs and  `iss` & `kid`.

Reply:
Created https://github.com/w3c/vc-jwt/issues/106